static/app/views/issueDetails/groupTags/groupTagsDrawer.tsx

```diff
@@ -186,37 +247,44 @@ export function GroupTagsDrawer({group}: {group: Group}) {
                 </CrumbContainer>
               ),
             },
-            {
-              label: t('All Tags'),
-              to: tagKey
-                ? {
-                    pathname: `${baseUrl}${TabPaths[Tab.TAGS]}`,
-                    query: location.query,
-                  }
-                : undefined,
-            },
-            ...(tagKey ? [{label: tagKey}] : []),
+            ...(tab === TAGS_TAB
+              ? [
+                  {
+                    label: t('All Tags'),
+                    to: tagKey
+                      ? {
+                          pathname: `${baseUrl}${TabPaths[Tab.TAGS]}`,
+                          query: location.query,
+                        }
+                      : undefined,
+                  },
+                  ...(tagKey ? [{label: tagKey}] : []),
+                ]
+              : []),
+            ...(tab === FEATURE_FLAGS_TAB
+              ? [
+                  {
+                    label: t('All Feature Flags'),
+                  },
+                ]
+              : []),
```

Comment:
This is pretty hard to read IMO

You can simplify it a bit...

```suggestion
            ...(tab === TAGS_TAB
              ? [
                  {
                    label: t('All Tags'),
                    to: tagKey
                      ? {
                          pathname: `${baseUrl}${TabPaths[Tab.TAGS]}`,
                          query: location.query,
                        }
                      : undefined,
                  },
                  ...(tagKey ? [{label: tagKey}] : []),
                ]
              : tab === FEATURE_FLAGS_TAB
              ? [
                  {
                    label: t('All Feature Flags'),
                  },
                ]
              : []),
```

but it might be even better if you pull this out and generate the crumbs not inline